integration/emr/alluxio-presto.json

```diff
@@ -0,0 +1,40 @@
+[
+  {     
+    "Classification": "hive-site",
+    "Properties": {
+      "fs.alluxio.impl": "alluxio.hadoop.FileSystem",
+      "fs.AbstractFileSystem.alluxio.impl": "alluxio.hadoop.AlluxioFileSystem",
+      "alluxio.user.file.writetype.default": "CACHE_THROUGH",
```

Comment:
is this write type required?

Reply:
Since we don't maintain the journal between runs, we want any created directories to be persisted to the UFS. Without a THROUGH or CACHE_THROUGH writetype, we would not only lose the data but also the directory structure on which the tables are defined.